libosmscout-import/src/osmscout/import/GenWayWayDat.cpp

```diff
@@ -34,6 +32,8 @@
 #include <osmscout/import/RawWay.h>
 #include <osmscout/import/Preprocess.h>
 
+#include <algorithm>
```

Comment:
We have are using little bit different style at work: own, 3rd party, stl... Because when you forgot to add some include in header file (`string` for example) but adds it to cpp file, you will not be noticed, when it is before your headers... 

But order you mention should work too. I will try to keep it in mind.